cmd/sync.go

```diff
@@ -99,6 +99,30 @@ type rawSyncCmdArgs struct {
 	// of the target traverser speed, so that the scanned information can then be used for ETA estimation.
 	//
 	maxObjectIndexerMapSizeInGB string
+
+	// Change file detection flag. Valid Flag are TargetCompare, Ctime, CtimeMtime.
+	// TargetCompare - Default sync comparsion where target enumerated for each file. It's least optimized, but gurantee no data loss.
+	// Ctime - CTime used to detect change files/folder. Should be used where mtime not reliable.
+	// CtimeMtime - Both CTime      and MTime used to detect changed files/folder. It's most efficient in all of the cfdModes.
+	cfdMode string
+
+	// In case of metadata change only, shall we transfer whole file or only metadata. This flag governs that.
+	metaDataOnlySync bool
+
```

Comment:
I have commented to add comments in syncDestinationComparator.
Instead, let's add all the comments here and refer all other places to here, instead of repeating the same comments in all places